examples/tv-casting-app/android/App/app/src/main/jni/com/matter/casting/support/IdentificationDeclarationOptions.java

```diff
@@ -0,0 +1,108 @@
+/**
+ * Copyright (c) 2024 Project CHIP Authors All rights reserved.
+ *
+ * <p>Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+ * except in compliance with the License. You may obtain a copy of the License at
+ *
+ * <p>http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * <p>Unless required by applicable law or agreed to in writing, software distributed under the
+ * License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.matter.casting.support;
+
+import android.util.Log;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * This class contains the optional parameters used in the IdentificationDeclaration Message, sent
+ * by the Commissionee to the Commissioner. The options specify information relating to the
+ * requested UDC commissioning session.
+ */
+public class IdentificationDeclarationOptions {
+  static final String TAG = IdentificationDeclarationOptions.class.getSimpleName();
+  private static final int CHIP_DEVICE_CONFIG_UDC_MAX_TARGET_APPS = 10;
+
+  public IdentificationDeclarationOptions() {}
+
+  /**
+   * Feature: Target Content Application Flag to instruct the Commissioner not to display a Passcode
+   * input dialog, and instead send a CommissionerDeclaration message if a commissioning Passcode is
+   * needed.
+   */
+  public boolean mNoPasscode = false;
```

Comment:
Avoid using C/C++ conventions like adding the m prefix in languages that don't need it like Java

Reply:
removing m prefix